benchto-driver/src/main/java/io/trino/benchto/driver/listeners/BenchmarkServiceExecutionListener.java

```diff
@@ -167,6 +171,38 @@ private void checkClocksSync()
                 });
     }
 
+    @Override
+    public Future<?> concurrencyTestExecutionFinished(List<QueryExecutionResult> executions)
+    {
+        return taskExecutor.submit(() -> {
+            FinishRequest finishRequest = new FinishRequestBuilder()
+                    .withStatus(ENDED)
+                    .withEndTime(
+                            executions.stream()
+                                    .map(e -> e.getUtcEnd().toInstant())
+                                    .max(Comparator.comparing(Instant::toEpochMilli))
+                                    .orElseThrow(NoSuchElementException::new))
+                    .addMeasurement(Measurement.measurement(
+                            "queries_successful",
+                            "NONE",
```

Comment:
`QUERY_COUNT` instead of `NONE`

Reply:
There's no such unit as QUERY_COUNT (they are defined in benchto-service/src/main/java/io/trino/benchto/service/model/MeasurementUnit.java)